### DISCUSSION TITLE
Buildable libs in watch mode when serve an app locally

### DISCUSSION THREAD

**Discussion by ihor-panasiuk95** [nrwl/nx#10917] [General]
There is one limitation with buildable libs that prevents buildable libs to be in watch mode while starting application in watch mode.
Is that possible to somehow make buildable lib to work as not-buildable when I start application in serve mode locally? With that I'll be able to get updates in browser when I change something in buildable lib code. I thought something about skipping builds for buildable libs and without tsconfig being replaced with one that points to buildable libs build artifacts that would allow to build libs in the scope of an application (not like buildable lib, when it's built separatly, and just consumed by an app). 
Is that possilbe? Or is there a way to workaround that so I can see updates in browser when I change code for buildable libs?

**Comment by jebesampaio** [comment]
This seems to be working out of the box...if you serve some app which is consuming some monorepo lib, it automatically triggers the reload.

**Reply by ihor-panasiuk95** [reply]
Only if that lib is not buildable. If the lib is buildable, it doesn't.

**Comment by dstoyanoff** [comment]
Hey @ihor-panasiuk95, did you figure this out? I am, like you, trying to find a way to provide hot reload of buildable libraries without luck for now. 

Thanks

**Comment by charlesbodman** [comment]
Same thing here. Any solutions?
